src/contractTest/java/uk/gov/hmcts/reform/professionalapi/provider/ProfessionalUserInternalControllerV2ProviderTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.professionalapi.provider;
+
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.ContextConfiguration;
+import uk.gov.hmcts.reform.professionalapi.WebMvcProviderTest;
+import uk.gov.hmcts.reform.professionalapi.configuration.WebConfig;
+import uk.gov.hmcts.reform.professionalapi.controller.internal.ProfessionalUserInternalControllerV2;
+import uk.gov.hmcts.reform.professionalapi.domain.ContactInformation;
+import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
+import uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus;
+import uk.gov.hmcts.reform.professionalapi.domain.ProfessionalUser;
+import uk.gov.hmcts.reform.professionalapi.repository.ProfessionalUserRepository;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.professionalapi.pact.util.PactUtils.getUserConfiguredAccesses;
+
+@Provider("referenceData_professionalInternalUsersV2")
+@WebMvcTest({ProfessionalUserInternalControllerV2.class})
+@AutoConfigureMockMvc(addFilters = false)
+@ContextConfiguration(classes = {ProfessionalUserInternalControllerV2ProviderTestConfiguration.class, WebConfig.class})
+public class ProfessionalUserInternalControllerV2ProviderTest extends WebMvcProviderTest {
+
+    @Autowired
+    ProfessionalUserRepository professionalUserRepository;
+
+    public static final String ORG_NAME = "Org-Name";
+    public static final String SRA_ID = "sra-id";
```

Comment:
do we need to define these public?

Reply:
probably not. I'll change the visibility modifier